dhall/src/Dhall/Set.hs

```diff
@@ -66,3 +67,6 @@ empty = Set Data.Set.empty Data.Sequence.empty
 difference :: Ord a => Set a -> Set a -> [a]
 difference os (Set s _) =
     filter (\ x -> not (Data.Set.member x s)) (toList os)
+
+sort :: Ord a => Set a -> Set a
```

Comment:
Could you add a haddock comment since it's part of the public API?

Reply:
Done!